ValidationRules.Storage/Model/Facts/Position.cs

```diff
@@ -1,51 +1,56 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 
 namespace NuClear.ValidationRules.Storage.Model.Facts
 {
     public sealed class Position
     {
-        public const long CategoryCodeSelfAdvertisementOnlyOnPc = 287; // Самореклама только для ПК
-        public const long CategoryCodeAdvertisementInCategory = 38; // Объявление в рубрике(Объявление под списком выдачи)
+        public const long CategoryCodeAdvertisementInCategory = 38; // Объявление в рубрике (Объявление под списком выдачи)
+        public const long CategoryCodeBasicPackage = 303; // пакет "Базовый" ???
+        public const long CategoryCodeMediaContextBanner = 395122163464046280; // Баннер в поисковой выдаче (онлайн-версия)
+        public const long CategoryCodeContextBanner = 809065011136692318; // Баннер в поисковой выдаче (мобильная версия)
 
-        public const long CategoryCodeBasicPackage = 303; // пакет "Базовый"
-        public const long CategoryCodeMediaContextBanner = 395122163464046280; // МКБ
-        public const long CategoryCodeContextBanner = 809065011136692318; // КБ
-        public const long CategoryCodePremiumPartnerAdvertising = 809065011136692321; // Реклама в профилях партнеров (приоритетное размещение)
-        public const long CategoryCodeCallToAction = 809065011136692314; // Кнопка-действие
-        public const long CategoryCodePartnerAdvertisingAddress = 809065011136692326; // Реклама в профилях партнеров (адреса)
+        public const long CategoryCodePremiumPartnerAdvertising = 809065011136692321; // Реклама компании в карточках партнёров (партнеры FMCG)_
+        public const long CategoryCodePremiumPartnerAdvertising2 = 809065011136692333; // Реклама компании в карточках партнёров (партнеры FMCG)
+        public const long CategoryCodePartnerAdvertisingAddress = 809065011136692326; // Реклама компании в карточках партнёров (адреса)
 
-        public const int BindingObjectTypeCategoryMultipleAsterix = 1;
+        public const int BindingObjectTypeCategoryMultipleAsterisk = 1;
         public const int BindingObjectTypeAddressMultiple = 35;
 
         public const int PositionsGroupMedia = 1;
 
         public const int ContentSalesWithoutContent = 1;
         public const int ContentSalesContentIsNotRequired = 2;
 
-        public static readonly IReadOnlyCollection<long> CategoryCodesAllowNotLocatedOnTheMap = new long[]
-            {
-                11, // Рекламная ссылка
-                14, // Выгодные покупки с 2ГИС
-                26, // Комментарий к адресу
-                809064675575595276, // Реклама в профилях партнеров (адрес)
-            };
+        public static readonly IReadOnlyCollection<long> CategoryCodesAllowNotLocatedOnTheMap = new[]
+        {
+            11, // Рекламная ссылка
+            14, // Выгодные покупки с 2ГИС
+            26, // Комментарий к адресу
+            CategoryCodePartnerAdvertisingAddress
```

Comment:
Идентификатор был 809064675575595276, теперь 809065011136692326 - так и задумано?

Reply:
Ага, я в описании написал, что я пофиксил старый баг, Андрей вместо Position.CategoryCode записал Position.Id в константу.
Я вернул как надо было изначально.

Reply:
можно комент добавить, что бы не смотреть в описание константы.
```suggestion
            CategoryCodePartnerAdvertisingAddress // Реклама компании в карточках партнёров (адреса)
```

Е = Единообразие.
ниже по тексту то же самое. в общем случае непонятно, почему некоторые long'и заслужили право быть константами, а другие - нет. это странно.

Reply:
Всем номерным номенклатурам дал символьные имена для единообразия